pyramid_oereb/standard/create_standard_models.py

```diff
@@ -39,12 +48,21 @@ def create_standard_model():
         type='string',
         help='The schema name. If not specified, it will be derived from the code.'
     )
+    parser.add_option(
+        '-s', '--primary_key_is_string',
+        dest='primary_key_is_string',
+        metavar='PRIMARY KEY IS STRING',
```

Comment:
This should be in one word or separate with _ with space it usually mean that we have 4 parameters...